<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>607</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>FEEDBACK MANAGEMENT SYSTEM</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">
      /* Style for all QPushButtons */
      QPushButton {
         background-color: #3498db;
         color: white;
         border: none;
         border-radius: 10px;
         padding: 8px;
         min-height: 50px;
         font-size: 16px;
      }
      QPushButton:hover {
         background-color: #2980b9;
      }
      /* Style for QLineEdits */
      QLineEdit {
         padding: 4px;
         min-height: 40px;
         font-size: 16px;
      }
    </string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="labelTitle">
      <property name="palette">
       <palette>
        <active>
         <colorrole role="WindowText">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>85</red>
            <green>255</green>
            <blue>255</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Text">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>85</red>
            <green>170</green>
            <blue>255</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="ButtonText">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>0</red>
            <green>128</green>
            <blue>0</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="LinkVisited">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>209</red>
            <green>130</green>
            <blue>255</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="PlaceholderText">
          <brush brushstyle="SolidPattern">
           <color alpha="128">
            <red>0</red>
            <green>128</green>
            <blue>0</blue>
           </color>
          </brush>
         </colorrole>
        </active>
        <inactive>
         <colorrole role="WindowText">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>85</red>
            <green>255</green>
            <blue>255</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Text">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>85</red>
            <green>170</green>
            <blue>255</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="ButtonText">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>0</red>
            <green>128</green>
            <blue>0</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="LinkVisited">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>209</red>
            <green>130</green>
            <blue>255</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="PlaceholderText">
          <brush brushstyle="SolidPattern">
           <color alpha="128">
            <red>0</red>
            <green>128</green>
            <blue>0</blue>
           </color>
          </brush>
         </colorrole>
        </inactive>
        <disabled>
         <colorrole role="WindowText">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>0</red>
            <green>128</green>
            <blue>0</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Text">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>0</red>
            <green>128</green>
            <blue>0</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="ButtonText">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>0</red>
            <green>128</green>
            <blue>0</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="LinkVisited">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>209</red>
            <green>130</green>
            <blue>255</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="PlaceholderText">
          <brush brushstyle="SolidPattern">
           <color alpha="128">
            <red>0</red>
            <green>128</green>
            <blue>0</blue>
           </color>
          </brush>
         </colorrole>
        </disabled>
       </palette>
      </property>
      <property name="font">
       <font>
        <family>Stencil</family>
        <pointsize>24</pointsize>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: ;</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::Shape::Box</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Shadow::Plain</enum>
      </property>
      <property name="lineWidth">
       <number>10</number>
      </property>
      <property name="text">
       <string>FEEDBACK MANAGEMENT SYSTEM</string>
      </property>
      <property name="textFormat">
       <enum>Qt::TextFormat::RichText</enum>
      </property>
      <property name="alignment">
       <set>Qt::AlignmentFlag::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="labelEmail">
      <property name="font">
       <font>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>ENTER YOUR EMAIL ADDRESS</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="emailLineEdit"/>
    </item>
    <item>
     <widget class="QLabel" name="labelUsername">
      <property name="font">
       <font>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>ENTER YOUR USERNAME</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="usernameLineEdit"/>
    </item>
    <item>
     <widget class="QPushButton" name="acceptButton">
      <property name="text">
       <string>ACCEPT AND CONTINUE</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="labelFeedback">
      <property name="font">
       <font>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>SUBMIT YOUR FEEDBACK</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="feedbackLineEdit"/>
    </item>
    <item>
     <widget class="QPushButton" name="submitFeedbackButton">
      <property name="text">
       <string>SUBMIT FEEDBACK</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="viewNegativeButton">
        <property name="text">
         <string>VIEW NEGATIVE FEEDBACK</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="viewAllButton">
        <property name="text">
         <string>VIEW ALL FEEDBACK</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>900</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
